src/brs/services/impl/BlockServiceImpl.java

```diff
@@ -187,12 +187,16 @@ public void apply(Block block) {
 
   @Override
   public long getBlockReward(Block block) {
-    if (block.getHeight() == 0 || block.getHeight() >= 1944000) {
-      return 0;
-    }
-    int month = block.getHeight() / 10800;
-    return BigInteger.valueOf(10000).multiply(BigInteger.valueOf(95).pow(month))
-        .divide(BigInteger.valueOf(100).pow(month)).longValue() * Constants.ONE_BURST;
+    return getBlockReward(block.getHeight());
+  }
+
+  public static long getBlockReward(int height) {
```

Comment:
not sure why you extracted this here...

Reply:
This one because I need the block reward having only the height on another place.